cel/io.go

```diff
@@ -120,3 +130,178 @@ func AstToString(a *Ast) (string, error) {
 	info := a.SourceInfo()
 	return parser.Unparse(expr, info)
 }
+
+// RefValueToValue converts between ref.Val and Value.
+// The ref.Val must not be error or unknown.
+func RefValueToValue(res ref.Val) (*exprpb.Value, error) {
```

Comment:
Don't forget to add some basic tests of conversion back and forth in the `io_test.go`

Reply:
Added a test in io_test.go